config/webpack.config.js

```diff
@@ -237,6 +237,7 @@ const pluginConfig = function(plugin) {
   configModules.unshift(path.resolve(__dirname, '..', 'node_modules'));
```

Comment:
Another thought: is this wrong?

Reply:
this adds `./usr/share/foreman/node_modules` (which is a symlink to `/usr/lib/node_modules`) so that the "toplevel" modules can be found, but then it doesn't find the modules inside the modules (turtles)